git/intermediate_git/a_deeper_look_at_git.md

```diff
@@ -96,7 +96,7 @@ squash 92aa6f3 Create second file
 pick 05e5413 Create third file and create fourth file
 ~~~
 
-Rename the commit to `Create first and second file`, then finish the rebase. That's it! Run `git log` and see how the first two commits got squashed together.
+Rename the 1st commit message to `Create first and second file`, delete commit message #2, then finish the rebase. That's it! Run `git log` and see how the first two commits got squashed together.
```

Comment:
First change is definitely solid as it can be a bit confusing exiting the editor the first time. That said I don't think it's necessary to change the renaming the commit message sentence. That said, since when we are squashing we are combining the commits rather than deleting or replacing one. I do feel it would be better to keep it as is or just have the "Rename the commit" be "Change the commit message to". As that would capture the steps you're mentioning in essence rather than just applying the steps narrowly.

Reply:
I see what you're saying about how I'm applying the steps narrowly and that implies we're deleting something. 

Changing the message of the newly squashed commit is definitely what I'm trying to convey here as it's not obvious how one would go about doing that. Given the instruction, one would assume that it becomes a single commit and we rename it in the same fashion as earlier in the lesson; however, the below editor is prompted after we exit the rebase tool changing pick to squash:
![image(1)](https://github.com/TheOdinProject/curriculum/assets/70310278/31a0a644-a371-4ce4-a3f1-8ed21f7a31b8)
Seeing the line "Please enter the commit message for your changes" led me to a bit of confusion on how I would go about that correctly.

Reply:
@EthanLeong97 Gotcha. The editor does explain how to make the changes a bit. If you have a different idea on how to rephrase it to make it clear were just replacing the commit message im more than happy to hear it. Maybe something like change "Rename the commit to'" with "Replace the old commit messages with". As it does become one commit and were just replacing the commit message. Kind of what we want to get at(In this section) is to explain the concept of what the person is doing rather than just the specific situational steps. Whereas in the other part the PR touched Git has very specific steps to exit the editor.